Update start-working-in-bcgov-github-organization.md (#32)
bcgov/bcdg: docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md

If you are intending to maintain an active project, make sure to establish the appropriate processes and terms to manage contributions.

## Integrating GitHub Apps

> We have large volumes of asks about adding GitHub Apps to our organization’s repository. To manage the asks, we created a request process through the devops-requests repository.

Reviewer: please link to the devops request repository here.

Reviewer 2: Here is the link to the specific DevOps request for requesting a GitHub App. I've reworked the sentence to use the link. Please feel free to revert back to previous sentence.

```suggestion
We have large volumes of asks about adding GitHub Apps to our organization’s repository. Request a GitHub App to be installed from the BCDevOps Request Issue: [Request for integrating a GitHub App](https://github.com/BCDevOps/devops-requests/issues/new?assignees=MonicaG%2C+oomIRL%2C+SHIHO-I&labels=github-app%2C+pending&projects=&template=github_integration_request.md&title=).
```

> If the existing applications, like SonarCloud, does not work for you or your team, please create an issue. We need: 

Reviewer: we've recently decided not to keep using sonar cloud. @MonicaG is there a better example of one we get requests for commonly?

Reviewer 2: Maybe just take the example out?

Reviewer 2: ```suggestion
If the existing applications don't work for you or your team, please create an issue. We need: 
```
